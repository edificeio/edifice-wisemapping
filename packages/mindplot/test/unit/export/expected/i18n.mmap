<?xml version="1.0" encoding="UTF-8"?><ap:Map xmlns:ap="http://schemas.mindjet.com/MindManager/Application/2003"><ap:OneTopic><ap:Topic OId="0W54nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="1W54nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Este es un é con acento"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="2W54nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Este es una ñ"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="3W54nezC90m8NYAi2fjQvw=="><ap:Text PlainText="這是一個樣本 Japanise。"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="i18n"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                        "/><ap:SubTopicsShape SubTopicsAlignment="urn:mindjet:Center" SubTopicsGrowth="urn:mindjet:Horizontal" SubTopicsGrowthDirection="urn:mindjet:AutomaticHorizontal&#10;                        " VerticalDistanceBetweenSiblings="150"/></ap:Topic></ap:OneTopic><ap:Relationships/></ap:Map>