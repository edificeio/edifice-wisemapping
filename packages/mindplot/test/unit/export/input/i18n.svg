<svg focusable="true" preserveAspectRatio="none" width="1440" height="900" viewBox="-277.95 -272.425 1224.00000 765.00000">
    <path style="fill:#495879 " stroke-width="1px" stroke="#495879" stroke-opacity="1" fill="#495879" fill-opacity="1" visibility="visible" d="M0.00,0.00 C19.77,0 39.53,52.00 59.30,52.00 39.53,55.00 19.77,5.00 0.00,7.00 Z"></path>
    <path style="fill:#495879 " stroke-width="1px" stroke="#495879" stroke-opacity="1" fill="#495879" fill-opacity="1" visibility="visible" d="M0.00,0.00 C19.77,0 39.53,-24.00 59.30,-24.00 39.53,-21.00 19.77,5.00 0.00,7.00 Z"></path>
    <path style="fill:#495879 " stroke-width="1px" stroke="#495879" stroke-opacity="1" fill="#495879" fill-opacity="1" visibility="visible" d="M0.00,0.00 C19.60,0 39.20,14.00 58.80,14.00 39.20,17.00 19.60,5.00 0.00,7.00 Z"></path>
    <g preserveAspectRatio="none" focusable="true" width="100" height="100" transform="translate(-28.50000,-18.50000) scale(1.00000,1.00000)" visibility="visible">
        <rect width="61" height="43" rx="6.45" ry="6.45" x="-2" y="-3" stroke-width="1px" stroke="rgb(241,163,39)" fill="rgb(244,184,45)" stroke-opacity="1" fill-opacity="1" style="cursor: default;"></rect>
        <rect width="57" height="37" rx="5.55" ry="5.55" x="0" y="0" stroke-width="2px" fill="rgb(80,157,192)" stroke="rgb(57,113,177)" style="cursor: default;"></rect>
        <text font-family="Verdana" font-size="13.4375" font-style="normal" font-weight="bold" fill="#ffffff" visibility="visible" style="cursor: default;" y="10" x="12">
            <tspan dy="1em" x="12">i18n</tspan>
        </text>
        <g preserveAspectRatio="none" focusable="true" width="0" height="20.8"></g>
    </g>
    <g preserveAspectRatio="none" focusable="true" width="100" height="100" transform="translate(58.50000,-14.00000) scale(1.00000,1.00000)" visibility="visible">
        <rect width="147" height="34" rx="5.1" ry="5.1" x="-2" y="-3" stroke-width="1px" stroke="rgb(241,163,39)" fill="rgb(252,235,192)" stroke-opacity="0" fill-opacity="0"></rect>
        <line stroke-width="1px" stroke="#495879" stroke-opacity="1" style="cursor: default;" x1="0" y1="28" x2="143" y2="28"></line>
        <text font-family="Verdana" font-size="10.75" font-style="normal" font-weight="normal" fill="rgb(82,92,97)" visibility="visible" style="cursor: move;" y="8" x="9">
            <tspan dy="1em" x="9">Este es un é con acento</tspan>
        </text>
        <ellipse width="6" height="6" rx="3" ry="3" cx="3" cy="3" stroke-width="1px" fill="rgb(224,229,239)" visibility="hidden" stroke="rgb(2,59,185)" style="cursor: default;"></ellipse>
        <g preserveAspectRatio="none" focusable="true" width="0" height="15.600000000000001"></g>
    </g>
    <g preserveAspectRatio="none" focusable="true" width="100" height="100" transform="translate(59.00000,-52.00000) scale(1.00000,1.00000)" visibility="visible">
        <rect width="92" height="34" rx="5.1" ry="5.1" x="-2" y="-3" stroke-width="1px" stroke="rgb(241,163,39)" fill="rgb(252,235,192)" stroke-opacity="0" fill-opacity="0"></rect>
        <line stroke-width="1px" stroke="#495879" stroke-opacity="1" style="cursor: default;" x1="0" y1="28" x2="88" y2="28"></line>
        <text font-family="Verdana" font-size="10.75" font-style="normal" font-weight="normal" fill="rgb(82,92,97)" visibility="visible" style="cursor: move;" y="8" x="9">
            <tspan dy="1em" x="9">Este es una ñ</tspan>
        </text>
        <ellipse width="6" height="6" rx="3" ry="3" cx="3" cy="3" stroke-width="1px" fill="rgb(224,229,239)" visibility="hidden" stroke="rgb(2,59,185)" style="cursor: default;"></ellipse>
        <g preserveAspectRatio="none" focusable="true" width="0" height="15.600000000000001"></g>
    </g>
    <g preserveAspectRatio="none" focusable="true" width="100" height="100" transform="translate(59.00000,24.00000) scale(1.00000,1.00000)" visibility="visible">
        <rect width="144" height="34" rx="5.1" ry="5.1" x="-2" y="-3" stroke-width="1px" stroke="rgb(241,163,39)" fill="rgb(252,235,192)" stroke-opacity="0" fill-opacity="0"></rect>
        <line stroke-width="1px" stroke="#495879" stroke-opacity="1" style="cursor: default;" x1="0" y1="28" x2="140" y2="28"></line>
        <text font-family="Verdana" font-size="10.75" font-style="normal" font-weight="normal" fill="rgb(82,92,97)" visibility="visible" style="cursor: move;" y="8" x="9">
            <tspan dy="1em" x="9">這是一個樣本 Japanise。</tspan>
        </text>
        <ellipse width="6" height="6" rx="3" ry="3" cx="3" cy="3" stroke-width="1px" fill="rgb(224,229,239)" visibility="hidden" stroke="rgb(2,59,185)" style="cursor: default;"></ellipse>
        <g preserveAspectRatio="none" focusable="true" width="0" height="15.600000000000001"></g>
    </g>
</svg>