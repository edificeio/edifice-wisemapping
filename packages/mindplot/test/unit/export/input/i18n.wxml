<?xml version="1.0" encoding="UTF-8"?>
<map name="basic" version="tango">
    <topic central="true" id="0" position="0,0" shape="rounded rectangle">
        <text><![CDATA[i18n]]></text>
        <topic id="1" order="0" position="200,0" shape="line">
            <text><![CDATA[Este es un é con acento]]></text>
        </topic>
        <topic id="2" order="0" position="-200,0" shape="line">
            <text><![CDATA[Este es una ñ]]></text>
        </topic>
        <topic id="3" order="4" position="200,100" shape="line">
            <text><![CDATA[這是一個樣本 Japanise。]]></text>
        </topic>
    </topic>
</map>
