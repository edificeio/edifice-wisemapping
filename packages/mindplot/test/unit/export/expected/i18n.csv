i18n 
,
            Este es un é con acento 
,
            Este es una ñ 
,
            這是一個樣本 Japanise。 
